src/libostree/ostree-diff.c

```diff
@@ -26,6 +26,16 @@
 #include "ostree-repo-private.h"
 #include "otutil.h"
 
+/* See ostree-repo.c for a bit more info about these ABI checks */
```

Comment:
Can you add a mention here and in `ostree-repo.c` that the holes were found with `pahole`?

Reply:
I like adding the info but how about just canonically in one place?  Fixup ⬇️